submariner-operator/submariner-metrics-redesign.md

```diff
@@ -0,0 +1,163 @@
+# Submariner metrics redesign
+
+Related Issue:
+[Exporting Submariner metrics without needing to open ports](https://github.com/submariner-io/submariner/issues/1944)
+
+## Summary
+
+Submariner Gateway and Globanet pods run with HostNetworking. To access metrics exported by these two we need to open ports
+in the infra firewall. Current design has a limitation where these ports might be in use by some other service,
+causing a conflict and metrics becoming inaccessible. With the move towards minimizing changes needed in cloud infra for
+submariner, there is a use case to export metrics in a way that doesn't require any of these ports to be opened.
```

Comment:
Port conflicts are an additional concern, discuss them last so that the infrastructure considerations aren’t split up.

```suggestion
Submariner Gateway and Globanet pods run with HostNetworking.
To access metrics exported by these two we need to open ports in the infra firewall.
With the move towards minimizing changes needed in cloud infra for Submariner,
there is a use case to export metrics in a way that doesn't require any of these ports to be opened.
The current design also has a limitation where these ports might be in use by some other service,
causing a conflict and resulting in metrics becoming inaccessible. 
```